lisa/tools/kdump.py

```diff
@@ -98,12 +98,15 @@ def _install_from_src(self) -> None:
         kexec_tar = wget.get(self._kexec_repo, str(tool_path))
         tar = self.node.tools[Tar]
         tar.extract(kexec_tar, str(tool_path))
-        kexec_source = tar.get_root_folder(kexec_tar)
-        code_path = tool_path.joinpath(kexec_source)
+        find_tool = self.node.tools[Find]
+        kexec_source_folder = find_tool.find_files(
+            tool_path, name_pattern="kexec-tools*", find_type="d"
+        )
+        code_path = tool_path.joinpath(kexec_source_folder[0])
         self.node.tools[Gcc]
         make = self.node.tools[Make]
-        self.node.execute("./configure", cwd=code_path).assert_exit_code()
-        make.make_install(cwd=code_path)
+        self.node.execute("./configure", cwd=code_path, sudo=True).assert_exit_code()
```

Comment:
You can use `expected_exit_code` and `expected_exit_code_failure_message` arguments in `node.execute` rather than `assert_exit_code()`

Reply:
Have added.